controllers/cache_handler.go

```diff
@@ -256,6 +260,10 @@ func (h DisruptionTargetWatcherHandler) findNotifiableEvents(eventsToSend map[st
 					"message", event.Message,
 					"timestamp", event.LastTimestamp.Time.Unix(),
 				)
+			} else if event.Reason == "Killing" && strings.Contains(event.Message, "Stopping container") && eventsToSend[chaosv1beta1.EventContainerWarningState] {
```

Comment:
a bit of a rant here: we have a leaky abstraction from K8s here. What better should be to define our own `Reason` enum and have another Go class to map from *K8s Reason to our own Reason*. Beside, we should less rely on string matching as these values are not in our control.

I suppose fixing this is out of scope of this PR, but hope to raise the point and hear your opinion.

Reply:
I agree with you, we can build a non-exhaustive list of all k8s events